cmd/agent/app/run.go

```diff
@@ -347,6 +347,11 @@ func StartAgent() error {
 		log.Errorf("Error while starting GUI: %v", err)
 	}
 
+	// Verify 'DD_URL' and 'DD_DD_URL' conflicts
+	if util.EnvVarAreSetAndNotEqual("DD_DD_URL", "DD_URL") {
+		log.Warnf("'DD_URL' and 'DD_DD_URL' variables are both set in environment. Using 'DD_DD_URL' value")
+	}
+
```

Comment:
Done in [f8052ef6](https://github.com/DataDog/datadog-agent/pull/11646/commits/f8052ef63ccba845e4fe81818a1cfb974ebbd841)

I moved the util function to `config.go` because I had import cycles and because functions called in `load()` are also in `config.go`